students/tlugosan/lesson03/mailroom.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+""" Automate a script to send a thank you mail, print a report of donor or quit the program."""
+
+table_dictionary = {
+    'Toni Orlando': [150.00, 200.00, 100.00],
+    'Amanda Clark': [1800.00],
+    'Robin Hood': [1234.56, 4500.34, 765.28],
+    'Gina Travis': [523.10, 75.00],
+    'Mark Johnson': [850.00, 20.14]
+}
+
+
+def select_action(actions):
+    """User selects an action by its corresponding order number."""
+    print('What would you like to do?')
+    for i, item in enumerate(actions):
+        print("{}) {}".format(i + 1, item))
+    choice_actions = input('Pick your selection by their corresponding number: ')
+    while True:
+        if not choice_actions.isnumeric() or int(choice_actions) not in range(1, len(actions) + 1):
```

Comment:
there are couple different ways to do.. but it will be cleaner when we get to the mailroom version that uses exception handling.

what you have is perfectly fine, another alternative:
`if not '1' < ans < str(len(options))`